torch/_inductor/ir.py

```diff
@@ -190,6 +190,42 @@ def ir_node_to_tensor(x, guard_shape=True):
     return t
 
 
+class OptionalAttr:
+    def __repr__(self):
+        return self.name
+
+
+class OptionalString(OptionalAttr):
+    def __init__(self):
+        self.name = "optional_string"
+
+
+class OptionalList(OptionalAttr):
+    def __init__(self):
+        self.name = "optional_list"
+
+
+class OptionalScalar(OptionalAttr):
+    def __init__(self):
+        self.name = "optional_scalar"
+
+
+def need_convert_to_optional(value):
+    return not value and V.graph.cpp_wrapper
```

Comment:
A simple condition, no need to have a dedicated function for it?

Reply:
Removed the function